docs/api/UserSettingsPlugin.md

```diff
@@ -47,43 +45,584 @@ org.rdk.UserSettings interface methods:
-| [GetPreferredClosedCaptionService](#GetPreferredClosedCaptionService) | Getting Preferred ClosedCaption Service |
-| [GetPrivacyMode](#GetPrivacyMode) | Getting Privacy Mode |
-
-
-<a name="SetAudioDescription"></a>
-## *SetAudioDescription*
+| [setAudioDescription](#method.setAudioDescription) | Setting Audio Description |
+| [setPreferredAudioLanguages](#method.setPreferredAudioLanguages) | Setting Preferred Audio Languages |
+| [setPresentationLanguage](#method.setPresentationLanguage) | Setting Presentation Languages |
+| [setCaptions](#method.setCaptions) | Setting Captions |
+| [setPreferredCaptionsLanguages](#method.setPreferredCaptionsLanguages) | Setting PreferredCaption Languages |
+| [setPreferredClosedCaptionService](#method.setPreferredClosedCaptionService) | Setting Preferred Closed Caption Service |
+| [setPinControl](#method.setPinControl) | Setting PinControl |
+| [setViewingRestrictions](#method.setViewingRestrictions) | Setting ViewingRestrictions |
+| [setViewingRestrictionsWindow](#method.setViewingRestrictionsWindow) | Setting viewingRestrictionsWindow |
+| [setLiveWatershed](#method.setLiveWatershed) | Setting LiveWatershed |
+| [setPlaybackWatershed](#method.setPlaybackWatershed) | Setting PlaybackWatershed |
+| [setBlockNotRatedContent](#method.setBlockNotRatedContent) | Setting BlockNotRatedContent |
+| [setPinOnPurchase](#method.setPinOnPurchase) | Setting setPinOnPurchase |
+| [getAudioDescription](#method.getAudioDescription) | Returns Audio Description |
+| [getPreferredAudioLanguages](#method.getPreferredAudioLanguages) | Returns Preferred Audio Languages |
+| [getPresentationLanguage](#method.getPresentationLanguage) | Getting Presentation Languages |
+| [getCaptions](#method.getCaptions) | Getting Captions Enabled |
+| [getPreferredCaptionsLanguages](#method.getPreferredCaptionsLanguages) | Getting Preferred Caption Languages |
+| [getPreferredClosedCaptionService](#method.getPreferredClosedCaptionService) | Getting Preferred ClosedCaption Service |
+| [getPinControl](#method.getPinControl) | Returns Pin Control |
+| [getViewingRestrictions](#method.getViewingRestrictions) | Returns Get Viewing Restrictions |
+| [getViewingRestrictionsWindow](#method.getViewingRestrictionsWindow) | Returns Get Viewing Restrictions Window |
+| [getLiveWatershed](#method.getLiveWatershed) | Returns Live Watershed |
+| [getPlaybackWatershed](#method.getPlaybackWatershed) | Returns Playback Watershed |
+| [getBlockNotRatedContent](#method.getBlockNotRatedContent) | Returns BlockNotRatedContent |
+| [getPinOnPurchase](#method.getPinOnPurchase) | Returns PinOnPurchase |
+
+
+<a name="method.setAudioDescription"></a>
+## *setAudioDescription [<sup>method</sup>](#head.Methods)*
 
 Setting Audio Description.
 
 ### Events
 
-No Events
+| Event | Description |
+| :-------- | :-------- |
+| [OnAudioDescriptionChanged](#event.OnAudioDescriptionChanged) | Triggered when the audio description changes. |
```

Comment:
This is still showing Pascal case, please change it to: onAudioDescriptionChanged